package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "build": "tsc",
     "package": "pkg .",
-    "test": "node --import tsx --test **/*.test.ts",
+    "test": "find tests -name '*.test.ts' | xargs node --import tsx --test",
```

Comment:
like the folder structure suggested; can we do this instead of recursively searching?
`node --import tsx --test tests/**/*.test.ts`